parcel.go

```diff
@@ -14,47 +14,89 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
+	res, err := s.db.Exec("insert into parcel (client, status, address, created_at) values (:Client, :Status, :Address, :CreatedAt)",
+		sql.Named("Client", p.Client),
+		sql.Named("Status", p.Status),
+		sql.Named("Address", p.Address),
+		sql.Named("CreatedAt", p.CreatedAt))
+	if err != nil {
+		return 0, err
+	}
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	lastId, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(lastId), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
+	res := s.db.QueryRow("select * from parcel where number = :number", sql.Named("number", number))
 	p := Parcel{}
-
+	// заполните объект Parcel данными из таблицы
+	err := res.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// реализуйте чтение строк из таблицы parcel по заданному client
 	// здесь из таблицы может вернуться несколько строк
-
+	query, err := s.db.Query("select * from parcel where client = :client", sql.Named("client", client))
+	if err != nil {
+		return nil, err
+	}
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
-
+	for query.Next() {
+		p := Parcel{}
+		query.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
```

Comment:
Лучше всегда проверять ошибки
```suggestion
		err = query.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
```